lib/lan_connection.js

```diff
@@ -12,7 +12,8 @@ mdns.excludeInterface('0.0.0.0');
 function LANConnection(opts) {
   this.auth = {};
   this.ip = Array.isArray(opts.addresses) ? opts.addresses[0] : opts.host;
-  this.auth.host = opts.host || '';
+  this.host = opts.host || '';
+  this.auth.host = this.ip;
```

Comment:
We should comment this more thoroughly, but it's the literal SSH connection object.

https://github.com/tessel/t2-cli/blob/master/lib/lan_connection.js#L75